Makefile

```diff
@@ -36,6 +38,15 @@ proto-gen-rust:
 	@if docker ps -a --format '{{.Names}}' | grep -Eq "^${containerProtoGenRust}$$"; then docker start -a $(containerProtoGenRust); else docker run  --name $(containerProtoGenRust) -v $(CURDIR):/workspace --workdir /workspace -d $(protoImageName) \
 		sh ./scripts/protocgen_rust.sh; fi
 
```

Comment:
Can we add a Optimize-all or build-all?

Reply:
I'll check in with  @sabinchitrakar  and try to migrate an optimized cosmwasm build to a docker container.